community-membership.md

```diff
@@ -71,10 +71,10 @@ Defined by: Member of the OpenTelemetry GitHub organization
     template](https://github.com/open-telemetry/community/blob/master/.github/ISSUE_TEMPLATE/membership.md))
   - Make sure that the list of contributions included is representative of your
     work on the project.
-- Have your sponsoring reviewers reply confirmation of sponsorship: `+1`
+- Have your sponsoring reviewers reply confirmation of sponsorship: `I support`
 - Once your sponsors have responded, your request will be reviewed by the
-  Governance Committee.  Any GC member can review the requirements and add
-  Members to the GitHub org. 
+  Technical Committee (TC).  Any TC member can review the requirements and add
```

Comment:
Why is the membership approver pool not being expanded rather than being changed from GC to TC? 
I would prefer this approver pool to be GC+TC rather than TC.

Reply:
my understanding this isn't about the approvals, but about actually adding new members to the org, which requires permissions that the GC does not need (legislative vs. executive branch analogy).

Reply:
Also the process is already defined by the GC and there is not too much to "approve", we just need to follow/enforce the rule.

Reply:
Let's do this right. I suggest setting up a GitHub Admin team (similar to how k8s does this). This GH admin team would take ownership and execution of adding new members to the project. Imo delegating this responsibility to the TC may not help evaluation velocity. @SergeyKanzhelev who is on the TC (and helped enormously in ensuring membership requests were processed quickly while on the GC) may still end up doing the work and may still result in a SPOF situation within the TC. 

I also think this is a great opportunity for other active contributors and members on the project to get more involved.

Reply:
@alolita can you clarify on the next steps you propose? I think creating the new "Admin" team will take discussions, changing the charter of the TC, etc., so probably we can start that process (GC has to decide) but I would not block this especially as you said @SergeyKanzhelev being the person who did the most work.

GC right now, by the charter, does not have admin rights in the org, so not sure adding GC is feasible right now. Maybe this can also be discussed.

Reply:
this PR is correcting factual issue. As per creating GitHub Admin team, if there are enthusiasts who are interested in this, we can initiate the process. Definitely there are many aspects of GitHub management that can be documented and improved. @alolita do you want to create a github issue to track the idea and seek for volunteers?